src/signals/settings/users/containers/Overview/components/DataView/components/DataViewBody/index.js

```diff
@@ -0,0 +1,60 @@
+import React, { useMemo } from 'react';
+import PropTypes from 'prop-types';
+
+import { StyledTR, StyledTD } from 'signals/settings/users/containers/Overview/components/DataView/styled';
+
+const DataViewBody = ({
+  data,
+  visibleColumns,
+  onItemClick,
+  primaryKeyColumn,
+  numberOfColumns,
+}) => {
+  const dataColumnsMissing = useMemo(
+    () => numberOfColumns - visibleColumns.length,
+    [numberOfColumns, visibleColumns.length]
+  );
+
+  return (
+    <tbody data-testid="dataViewBody">
+      {data.map(row => (
+        <StyledTR
+          key={JSON.stringify(row)}
+          data-item-id={primaryKeyColumn && row[primaryKeyColumn]}
+          onClick={onItemClick}
+          data-testid="dataViewBodyRow"
+        >
+          {visibleColumns.map(column => (
+            <StyledTD
+              key={JSON.stringify(column)}
```

Comment:
It could but if that happens we're also showing duplicate data...

Reply:
It could very well be the case that we're showing similar or duplicate data; in a list of users, a lot will have the same value for the column `status`.